app/code/Magento/Catalog/Controller/Adminhtml/Product/Initialization/Helper.php

```diff
@@ -119,6 +124,7 @@ public function __construct(
         \Magento\Catalog\Model\Product\Initialization\Helper\ProductLinks $productLinks,
         \Magento\Backend\Helper\Js $jsHelper,
         \Magento\Framework\Stdlib\DateTime\Filter\Date $dateFilter,
+        Escaper $escaper,
```

Comment:
Let's try to make this change as backward compatible as possible, adding new constructor parameter as **last** and **optional**, with property initialized with object manager if the parameter is not passed.

e.g.

    $this->escaper = $escaper ?: ObjectManager::getInstance()->get(Escaper::class);